src/GitVersion.LibGit2Sharp/Git/GitCache.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections.Concurrent;
+
+namespace GitVersion.Git;
+
+internal sealed class GitCache
```

Comment:
@pvanbuijtene can we move the GitCache into the GitRepository itself? The dictionaries and the methods, and then pass the repository to the constructors instead of the GitCache instance?

Reply:
Sure, it's done.